src/main/java/Duke/Ui.java

```diff
@@ -0,0 +1,148 @@
+     */
+    public void listValidDateFormats() {
+        System.out.println("These are the following accepted formats:");
+        System.out.println("1) yyyy-mm-dd");
+    }
+
+    /**
+     * Prints a message after marking a task as done.
+     */
+    public void markDone(String taskName) {
+        System.out.println("Nice! I have marked (" + taskName + ") as done!");
+    }
+
+    /**
+     * Prints a message after marking a task as undone.
+     */
+    public void markUndone(String taskName) {
+        System.out.println("Nice! I have marked (" + taskName + ") as undone!");
+    }
+
+    /**
+     * List all the tasks in the list.
+     *
+     * @param taskList task list that contains tasks to be displayed.
+     */
+    public void listTasks(TaskList taskList) {
+        StringBuilder output = new StringBuilder();
+        output.append("\nCurrent Tasking\n");
+        for (int i = 0; i < taskList.getNumberOfTask(); i++) {
+            output.append(String.format("%d) %s\n", i + 1, taskList.getTask(i)));
+        }
+        output.append("Number of tasking: ").append(taskList.getNumberOfTask());
+        System.out.println(output);
+    }
+
+    /**
+     * A function that list all the tasks in the list based on a search.
+     *
+     * @param taskList task list that contains tasks to be displayed.
+     * @param search   contains the string we are interested in.
+     */
+    public void findTasks(TaskList taskList, String search) {
+        StringBuilder output = new StringBuilder();
+        int count = 0;
+        output.append("\nSearch Results\n");
+        for (int i = 0; i < taskList.getNumberOfTask(); i++) {
+            Task task = taskList.getTask(i);
+            if (task.getTaskName().contains(search)) {
+                output.append(String.format("%d) %s\n", i + 1, task));
+                count += 1;
+            }
+        }
+        output.append("Number of tasking: ").append(count);
+        System.out.println(output);
+    }
+
+    /**
+     * Prints a message after marking adding a task.
+     *
+     * @param task the task that was added.
+     */
+    public void addTask(Task task) {
+        System.out.printf("Successfully added: %s", task.getTaskName());
+    }
+
+    /**
+     * Prints a message after marking deleting a task.
+     *
+     * @param task the task that was deleted.
+     */
+    public void deleteTask(Task task) {
+        System.out.printf("Successfully deleted: %s", task.getTaskName());
+    }
+
+    /**
+     * Prints a message that should be displayed before closing the application.
+     */
+    public void endPrompt() {
+        System.out.println("Goodbye");
+    }
+
+    /**
+     * Prints a line to divide outputs from the application.
+     */
+    public void lineDivider() {
+        System.out.println("-".repeat(60));
+    }
+
+    /**
+     * Prints a success message when a file is successfully loaded.
+     */
+    public void fileSuccessfullyLoaded() {
```

Comment:
Perhaps this can be written as `loadFileSuccessfully()` to match the coding standard given below?

> Names representing methods must be verbs and written in camelCase.

Reply:
Thank you for raising this point. Will amend accordingly!